scripts/split_repo_migration/multiple-library.post-process.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```

Comment:
Please could you add this file to the README?
https://github.com/googleapis/google-cloud-python/blob/main/scripts/split_repo_migration/README.rst

Reply:
Done. PTAL.